### ISSUE TITLE
Add dataset: Spanish_Golden_Age_Sonnets

### ISSUE THREAD

**Issue by thebooort** [bigscience-workshop/lam#42]
### A URL for this dataset

https://github.com/bncolorado/CorpusSonetosSigloDeOro

### Dataset description

This corpus comprises sonnets written in Spanish between the 16th and 17th centuries.
Each sonnet has been annotated in XML in accordance with the TEI standard. Besides the header and structural information, each sonnet includes the formal representation of each verse’s particular metrical pattern.
The dataset is very interesting both for analyzing patterns / writer's behaviors, but also to train models with historical language data. 
Also I am planning to add extra information (translating) on the dataset card related to the descriptions that are in Spanish.

### Dataset modality

Text

### Dataset licence

Creative Commons Attribution Non Commercial 4.0 International

### How can you access this data

As a download from a repository/website

### Confirm the dataset has an open licence

- [X] To the best of my knowledge, this dataset is accessible via an open licence

### Contact details for data custodian

https://data.cervantesvirtual.com/

**Comment by davanstrien** [comment]
@thebooort  thanks for suggesting this. Looks great. We're likely to have quite a few source datasets with TEI. I didn't look closely at the schema for this but one option that could be interesting to explore would be to support one config that is focused on capturing more of the metadata captured in TEI and one focused on the text. Feel free to shout if you decide to work on this and want any input.

**Comment by thebooort** [comment]
Yes! I will be working in it to add it. I will start reading all the docs on the academic paper they provide, and will move from there. Will ask u any doubt I found along the way ! :)

**Comment by shamikbose** [comment]
@thebooort Are you working on this dataset? I would like to add this dataset if you aren't working on it already

**Comment by thebooort** [comment]
Hi @shamikbose  I have a version of this dataset in huggingface (sorrry for the late reply!)
https://huggingface.co/datasets/thebooort/spanish_golden_age_sonnets
I added the datacard info too. so it may be ready for review! 
Also @davanstrien  let me know if I should change anything or make anything in a different way (should i transfer it to Biglam ORG already or wait till the corrections/changes?)

**Comment by thebooort** [comment]
#self-assign

**Comment by thebooort** [comment]
#ready-for-review

**Comment by davanstrien** [comment]
@thebooort, thanks for working on this! 

Happy for you to transfer this over to the BigLAM organization. I will make a pull request with some suggestions for the README, but this looks great otherwise.

**Comment by thebooort** [comment]
ty!  I just transferred the repository, let me know if everything is ok! (I see no preview for de dataset rn)

**Comment by davanstrien** [comment]
> ty! I just transferred the repository, let me know if everything is ok! (I see no preview for the dataset rn)

Thanks, I just made a small pull request with some suggested changes (https://huggingface.co/datasets/biglam/spanish_golden_age_sonnets/discussions/1). If this looks okay feel free to merge (or I can also do it).

**Comment by davanstrien** [comment]
@thebooort thanks again for working on this :)

**Comment by thebooort** [comment]
@davanstrien thanks to you and all the organisers of the hackathon! :smile:
